contracts/escrow/EscrowReward.sol

```diff
@@ -94,8 +99,7 @@ contract EscrowReward is Escrow {
 		uint256 reward = userBalances[msg.sender].mul(totalRewardDeposit).div(totalDeposit);
 		withdrawTokens();
 
-		bool txStatus = rewardToken.transfer(msg.sender, reward);
-		require(txStatus, "Token transfer was not successful. Check receiver address.");
+		lockedSOV.depositSOV(msg.sender, reward);
```

Comment:
function name and natspec don't fit the function's logic

Reply:
This is the NatSpec:

```
	/**
	 * @notice Withdraws token from the contract by User.
	 * @dev Only works after the contract state is in Withdraw.
	 */
```

Maybe I can try to explain a bit more regarding the LockedSOV part, is that what you meant?

Reply:
That's definitely subjective. When I see  `withdraw` I expect that caller receives assets. The code implements a more complex scheme